src/gql/queries/get-commit-queue.ts

```diff
@@ -1,5 +1,7 @@
 import { gql } from "@apollo/client";
+import { CodeChangesTable } from "gql/fragments/codeChangesTable";
 
+/* eslint-disable */
```

Comment:
can you disable the `graphql/template-strings` rule instead of eslint in general

Reply:
I was unable to target `graphql/template-strings` with a disable comment. I'm unsure why and didn't look into it too closely. I think this query file will become more ergonomic once we switch over to `.graphql` files